Gruntfile.js

```diff
@@ -138,6 +140,19 @@ module.exports = function (grunt) {
   const activeBrowser = grunt.option('bedrock-browser') || 'chrome-headless';
   const headlessBrowser = activeBrowser.endsWith("-headless") ? activeBrowser : 'chrome-headless';
   const activeOs = grunt.option('bedrock-os') || 'tests';
+
+  const remote = grunt.option('remote');
+
+  const bedrockOpts = (grunt, availableOpts) => {
+    return availableOpts.reduce((opts, opt) => {
```

Comment:
I went with `reduce` to return a new object that I'm passing as the new opts. I wanted to have a nice object to merge later on

Reply:
You can still do that, though. And this code is a lot more obvious about what's happening.

```
const bedrockOpts = {};
availableOpts.keys().forEach(opt => {
  const current = grunt.option(opt);
  if (current) bedrockOpts[opt] = current;
});
```

Reply:
I like the reduce cause you get back the object instead of changing an object outside the function. Happy to change it if it fits better in that style

Reply:
you are mutating an object that exists outside the function, though, you're just pretending you don't by making it the reduce argument 😛